### PR TITLE
Move CallAutomation GA3 Changes to ALPHA3

### PR DESCRIPTION
**Pull request by amariwest-msft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by minwoolee-msft** [comment]
I see changes for other ACS sdk changes - such as job router - should those be included here?

File: sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py [hunk 1]
```diff
@@ -489,7 +489,7 @@ async def start_recording(
          ``External`` enables bring your own storage.
         :paramtype recording_storage_type: str or None
         :keyword external_storage_location: The location where recording is stored,
-         when RecordingStorageType is set to 'BlobStorage'.
+         when RecordingStorageKind is set to 'BlobStorage'.
```
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py:492]: azure blob storage

Comment (amariwest-msft) [reply sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py]: replaced it with azureBlobStorage

File: sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_operations/_patch.py
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_operations/_patch.py:1]: Let's not include other communication SDK changes in this PR

**Review by minwoolee-msft — APPROVED** [review]

**Pull request merged**